packages/ListBox/src/ListBox.js

```diff
@@ -228,7 +228,7 @@ export const propTypes = {
   /** Callback returning the current selection on the ListBox */
   onChange: PropTypes.func,
 
-  /** Defaults label to display when the ListBox has not option selected */
+  /** Default label for trigger when the ListBox has no option selected */
```

Comment:
I did not include this change in the `changeset` as it has no effect on the actual code for the `ListBox`.